src/cascadia/TerminalApp/Profile.cpp

```diff
@@ -319,23 +320,25 @@ Profile Profile::FromJson(winrt::Windows::Data::Json::JsonObject json)
     {
         result._schemeName = json.GetNamedString(COLORSCHEME_KEY);
     }
-    else
+    else if (json.HasKey(COLORSCHEME_KEY_OLD))
+    {
+        // TODO: deprecate old settings key

```

Comment:
We may want to file a github issue for this and link it here

Reply:
```suggestion
        // TODO:GH#1069 deprecate old settings key
```